### ISSUE TITLE
categories

### ISSUE THREAD

**Issue by larshp** [larshp/abapOpenChecks#178]
add category for each check?

Obsolete, nicer code, reducing code

Perhaps ABBA encode the categories? Waterloo = old stuff, Name of the Game = obsolete, Dancing Queen = niceer code etc

**Comment by larshp** [comment]
Robustness
Security
Architecture
Compliance
Performance
Maintainability
Bug Risk
Clarity
Compatibility
Readability
Complexity
Duplication
Style

**Comment by DennstedtB** [comment]
I would move the unit test category to robustness. The rest would make sense to me

**Comment by larshp** [comment]
yeah, its just difficult to categorize all under the best fitting category, guess we will just have to try

**Comment by DennstedtB** [comment]
What about this topic? I guess we don't have that much time anymore - we already are at check 93. And I recently got some ideas for some new checks ^^ ( need to get a wording for it, then I'll creates issue + PR )
